package.json

```diff
@@ -25,5 +26,8 @@
     "sinon-chai": "^2.10.0",
     "source-map-support": "^0.4.15",
     "webpack": "^2.5.1"
+  },
+  "dependencies": {
+    "bluebird": "^3.5.0"
```

Comment:
The whole point of the configure approach was to avoid this dependency locking for libs with matching APIs